exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+const name = "Askin";
+const nameLength = name.length;
+console.log(`My name is ${name} and my name is ${nameLength} characters long.`);
```

Comment:
Well done for using this syntax to concatenate

Reply:
Hi Aisha, thank you very much for the feedback.